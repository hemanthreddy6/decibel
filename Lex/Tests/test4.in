import
const
load
save
play
function
if
or
otherwise
loop
over
long
int
float
string
audio
bool
true
false
continue
break
return
HIGHPASS
LOWPASS
EQ
SIN
COS
EXP_DECAY
LIN_DECAY
SQUARE
SAW
TRIANGLE
PAN
to