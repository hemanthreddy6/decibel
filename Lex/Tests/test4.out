IMPORT
CONST
LOAD
SAVE
PLAY
IDENTIFIER
IF
OR
OTHERWISE
LOOP
OVER
LONG
INT
FLOAT
STRING
AUDIO
BOOL
TRUE
FALSE
CONTINUE
BREAK
RETURN
HIGHPASS
LOWPASS
EQ
SIN
COS
EXP_DECAY
LIN_DECAY
SQUARE
SAW
TRIANGLE
PAN
TO
